server.js

```diff
@@ -34,19 +34,19 @@ let locales = ["sv-SE", "en-US"];
 class Metadata {
     constructor(file, cb) {
         let self = this;
-        this.cb = cb;  
+        this.cb = cb;
         this.db = {};
         this.last_updated = new Date();
         this.count = 0;
 
         if (args.redis) {
-            this.redis_idx = new redisIndexer();
-            this.redis_idx.create();
+            this.idx = new redisIndexer();
+            this.idx.create();
         } else {
-            this.lunr_idx = new lunrIndexer;
+            this.idx = new lunrIndexer;
```

Comment:
maybe lunrIndexer() ? Not sure if it matters but consistency is always good.

Reply:
That's right. Let me change that.